We have patched these modules into the leksah-server library.

--- haskell-leksah-0.10.0.4.orig/leksah.cabal
+++ haskell-leksah-0.10.0.4/leksah.cabal
@@ -99,7 +99,7 @@ library
                    bytestring >=0.9.0.1 && <0.10, containers >=0.2.0.0 && <0.5, directory >=1.0.0.2 && <3.1,
                    filepath >=1.1.0.1 && <1.3, glib >=0.10 && <0.13, gtk >=0.10 && <0.13,
                    gtksourceview2 >=0.10.0 && <0.13, mtl >=1.1.0.2 && <2.1, old-time >=1.0.0.1 && <1.1,
-                   parsec >=2.1.0.1 && <3.2, pretty >=1.0.1.0 && <1.1, process-leksah >=1.0.1.3 && <1.1,
+                   parsec >=2.1.0.1 && <3.2, pretty >=1.0.1.0 && <1.1,
                    regex-tdfa ==1.1.*, regex-base ==0.93.*, utf8-string >=0.3.1.1 && <0.4, array >=0.2.0.0 && <0.4,
                    time >=0.1 && <1.3, ltk >= 0.10 && <0.11, binary-shared >= 0.8 && <0.9, deepseq >= 1.1.0.0 && <1.2,
                    hslogger >= 1.0.7 && <1.2, leksah-server >= 0.10 && <0.11, network >= 2.2 && <3.0,
--- /dev/null
+++ haskell-leksah-0.10.0.4/patches/no-process-leksah.patch
@@ -0,0 +1,13 @@
+Index: haskell-leksah-0.10.0.4/leksah.cabal
+===================================================================
+--- haskell-leksah-0.10.0.4.orig/leksah.cabal	2011-05-27 20:23:58.000000000 +0200
++++ haskell-leksah-0.10.0.4/leksah.cabal	2011-05-27 20:24:02.000000000 +0200
+@@ -99,7 +99,7 @@
+                    bytestring >=0.9.0.1 && <0.10, containers >=0.2.0.0 && <0.5, directory >=1.0.0.2 && <3.1,
+                    filepath >=1.1.0.1 && <1.3, glib >=0.10 && <0.13, gtk >=0.10 && <0.13,
+                    gtksourceview2 >=0.10.0 && <0.13, mtl >=1.1.0.2 && <2.1, old-time >=1.0.0.1 && <1.1,
+-                   parsec >=2.1.0.1 && <3.2, pretty >=1.0.1.0 && <1.1, process-leksah >=1.0.1.3 && <1.1,
++                   parsec >=2.1.0.1 && <3.2, pretty >=1.0.1.0 && <1.1,
+                    regex-tdfa ==1.1.*, regex-base ==0.93.*, utf8-string >=0.3.1.1 && <0.4, array >=0.2.0.0 && <0.4,
+                    time >=0.1 && <1.3, ltk >= 0.10 && <0.11, binary-shared >= 0.8 && <0.9, deepseq >= 1.1.0.0 && <1.2,
+                    hslogger >= 1.0.7 && <1.2, leksah-server >= 0.10 && <0.11, network >= 2.2 && <3.0,
--- /dev/null
+++ haskell-leksah-0.10.0.4/patches/series
@@ -0,0 +1 @@
+no-process-leksah.patch
